README.md

```diff
@@ -61,7 +65,29 @@ will need to include the following dependencies at runtime yourself:
 - `@improbable-eng/grpc-web`
 - `browser-headers`
 
-See `//test:pizza_service_proto_test_suite` for an example.
+For generating grpc output files, you'll also need the following in your `BUILD` file:
+```python
+# For grpc-web support use:
+typescript_grpc_web_library(
+  name = "test_ts_grpc_web",
+  proto = ":test_proto",
+)
+
+# For grpc-node support use:
+typescript_grpc_node_library(
+  name = "test_ts_grpc_node",
+  proto = ":test_proto",
+)
+
+# For grpc-node support with grpc-js use:
+typescript_grpc_node_library(
+  name = "test_ts_grpc_node_grpc_js",
+  proto = ":test_proto",
+  mode = "grpc-js",
```

Comment:
I'm not a big fan of using strings as enums because they're prone to typos and don't have compile time checking.

Can we expose the knob a different way? Maybe with a separate macro for grpc-js, a boolean flag, or worse case also provide a constant that you can import to set this value.

(The internal string parameters to `typescript_proto_build` are fine because we're not exposing the API publicly)